examples/create_libraries/example/hw_example.dart

```diff
@@ -1,5 +1,5 @@
 import 'package:create_libraries/hw_mp.dart';
 
-main() {
+void main() {
```

Comment:
The analyzer was complaining about a missing return type, and we include the `void` return type for the `alarm()` method, so I added it here too.